### ISSUE TITLE
Localisation key "catalog.opds.info.availableSince" not working in Turkish

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2688]
From @catdotjs:

> This is impossible to translate. In turkish word "since"/"depuis" works according to date.
> So for example "2018'den beri" = "Since 2018", "2006'dan beri" = "Since 2006", and so on. 
> I need the date to be on the front or further context so I can substitute it.

**Comment by oguzugur28** [comment]
Hi All,
My native language ıs Turkish and today i've connected with EDRLab to translate the Thorium Reader to Turkish.

They've referred to this Github entry.

i understand the issue that you try to find a solution.

The issue that you're trying to solve it arise from Turkish language's structure.
Simply, Turkish structure is almost completely different from Indo-European Language family's language; verbs are located at end of the sentences, prepositions are located at end of the words etc.
Therefore, I think, we need to translate the interface line-by-line.
I can help you to translate the app. I can translate sentences or words. If you prefer, I can control translations which are already done.
Best,

**Comment by catdotjs** [comment]
@oguzugur28 we have a currently active repo in gitlocalize. I've done most of it, but if you want you can do some longer sentences. Please do read the translated words first, afterwards feel free to translates some of the sentences. 

https://gitlocalize.com/repo/9626/tr/src/resources/locales/en.json

I will be proof reading any of the translations so feel free to do the same for my translations.

**Comment by oguzugur28** [comment]
Hi @catdotjs ,
I understand you. If one of any translation is needed to change completely, I will inform about it.
I suppose that you know something about Turkish, therefore we can move forward easily.
Best,

**Comment by catdotjs** [comment]
@oguzugur28 yeah I'm fluent in Turkish due to... being Turkish, born and raised. In case of a translation being wrong or needing a correction, there should be a flag next-to each translation entry, you can use that to inform me if something is wrong.

**Comment by danielweck** [comment]
This is not something we can fix in Thorium 3.1, because there are several other similar cases in the localisation technique where we are not correctly handling plurals, ":" and "," punctuation joins, etc.

https://github.com/edrlab/thorium-reader/blob/1e2c98cbac4b5a248bcd3319c9a89887e3d1419a/src/renderer/library/components/dialog/publicationInfos/opdsControls/OpdsLinkProperties.tsx#L53

Related: https://github.com/edrlab/thorium-reader/pull/2743
